src/pybind/mgr/diskprediction_cloud/module.py

```diff
@@ -22,16 +22,38 @@
 ORIGIN_ALPHABET = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/"
 
 
+def get_transtable():
+    if sys.version_info[0] == 3 and sys.version_info[1] == 7:
```

Comment:
this is going to break for Python 3.6 and 3.8

Reply:
Agreed, this looks brittle.

Reply:
Thanks. Changed to compatible with Python 3.x